build/patches/Remove-navigator-connection-info.patch

```diff
@@ -0,0 +1,97 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Thu, 14 Jul 2022 09:48:45 +0000
+Subject: Remove navigator.connection info
+
+Change the result of navigator.connection to default values 
+and disable observers
+---
+ .../renderer/modules/netinfo/network_information.cc   |  7 ++++++-
+ .../renderer/modules/netinfo/network_information.h    |  1 +
+ .../platform/network/network_state_notifier.cc        | 11 +++++++++++
+ .../platform/network/network_state_notifier.h         |  2 +-
+ 4 files changed, 19 insertions(+), 2 deletions(-)
+
+diff --git a/third_party/blink/renderer/modules/netinfo/network_information.cc b/third_party/blink/renderer/modules/netinfo/network_information.cc
+--- a/third_party/blink/renderer/modules/netinfo/network_information.cc
++++ b/third_party/blink/renderer/modules/netinfo/network_information.cc
+@@ -61,7 +61,7 @@ NetworkInformation::~NetworkInformation() {
+ }
+ 
+ bool NetworkInformation::IsObserving() const {
+-  return !!connection_observer_handle_;
++  return !!connection_observer_handle_ || is_fake_observing_;
```

Comment:
Actually we just need to return always false here.

Reply:
no, it is necessary, otherwise all the `dchecks` fails.
it's just to minimize the changes